autonomic-administration-algorithms/src/main/java/br/com/autonomiccs/autonomic/administration/algorithms/impl/ConsolidationScoredPreferenceForSmallHosts.java

```diff
@@ -40,6 +40,11 @@
  */
 public class ConsolidationScoredPreferenceForSmallHosts extends ConsolidationAlgorithmBase {
 
+    /**
+     * This constant is the same as {@link ConsolidationAlgorithmBase#BYTES_TO_MEGA_BYTES}, but it is of type double.
+     */
+    protected final static double BYTES_TO_MEGA_BYTES_AS_DOUBLE = ConsolidationAlgorithmBase.BYTES_TO_MEGA_BYTES * 1d;
+
```

Comment:
Sure there is, and I forgot during the review of this component.
every division by int will result in an int.

Having said that, if a division is supposed to return 1.5, but the divisor is an int, it will return 1. That is why I tried to find a nice way to tranform that variable into a double type. I did not do that on the method because wanted to save some operations.